src/lib.rs

```diff
@@ -142,3 +142,6 @@ mod sealed {
     // ```
     pub trait Sealed {}
 }
+
+const PERLASM_CONFIGURED: () =
```

Comment:
This must be named `_PERLASM_CONFIGURED` to pass the `clippy` job.

Reply:
This assertion fails when running the `package` job.

Reply:
> This must be named `_PERLASM_CONFIGURED` to pass the `clippy` job.

Done.

Reply:
> This assertion fails when running the `package` job.

See below.